### ISSUE TITLE
[Suggestion]: More story mode accurate wither sickness

### ISSUE THREAD

**Issue by Wolfzul** [nonamecrackers2/crackers-wither-storm-mod#2216]
### Guidelines

- [X] By checking this box, I am saying that I have read the guidelines and believe to be following them to the best of my ability

### Description of feature request

So I was playing a little itty bit of Minecraft story mode recently, and just got to the part in episode 3 where you find Gabriel but he doesn't remember his name and all that because of wither sickness. I think you can tell where this is going.

**warning: I fucking yapped**
Oh yeah and just remember you're the dev, so add or don't add whatever you want about this suggestion. This is your mod.

Alright so this would probably be like a gamerule or something for players who don't want to deal with it, but basically, if you have wither sickness for too long/your maximum HP gets too low, you get wither sickness II or something.

Wither sickness II would add pink/magentaish or whatever screen effects to your screen, maybe with some small blurring or distortions, and sound might become a little distorted too.

Maybe there could be a feature where if your playing with other people their user names get censored, randomized or swapped with other players in the server, along with maybe their skins too. When censored, usernames would become blank and/or skins would become pitch black with white dots describing facial features.

There would probably be subgamerules, like disabling skin/username tampering, if you don't want to deal with those but want the other effects.

After a while, there could be a wither sickness III, that adds more audio/visual distortion, and skin/name distortion if enabled.
Maybe even slowness, simulating wither sickness in actual Minecraft storymode, when petra/Gabriel can't fight as hard, or move as fast. These weakening effects could probably be a subgamerule too.

Now obviously, if you were playing with a longtime friend on discord you wouldn't forget who they are, as you could probably just tell them to jump. So it would be cool if there was simple voice chat compatibility, where VC gets swapped with other players or heavily distorted.

Maybe you could add a feature to wither sickness III that distorts other players for you enough that they periodically are replaced with hostile mobs that only you can see, with your friend completely disappearing. While your friend is gone, the fake sickened mob would start to walk towards you, if you were hit by it, you would take no damage and your friend would reappear where they were.

Basically, schizophrenia. Your friend disappears, and a zombie starts moving towards you in their place, after a while or when you get hit by it, they reappear where they were, on like the other side of the room or something, as if nothing happened. If you have Simple Voicechat on, then you cannot hear you friends voice, only hearing zombie sounds, or very, very distorted sounds of your friends voice, that almost sound like whispering.

Anyways, hope you had fun reading my silly little yap session, I hope you consider some of these features. Remember, your mod, you don't have to add any of these features if you don't want to, and I'm fine with that. But thank you for reading my yap session, and hope you do add some of these features. byebye 👍

**Comment by watersofnazareth** [comment]
Well currently with Wither Sickness we don't want to make it too extreme since if you introduce new things it could ruin game progress, I will say I think your ideas are pretty good, having usernames scramble could be really fun, and an indicator that death is imminent could be a good idea, but with weakness/slowness it'd make game progression a nightmare, and we don't mind the yapping, giving us a good description is always good to do

**Comment by TaintedBlock** [comment]
Holy, this is a wall of text, lol. Though I love the ideas

**Comment by Herobrine982950291** [comment]
Isn't suggesting story-mode-ideas against the guidelines?

**Comment by TaintedBlock** [comment]
> Isn't suggesting story-mode-ideas against the guidelines?

Up to a certain extent. It would be a little different if the guy just straight up said, "make sickness more like story mode" and that's it

**Comment by Wolfzul** [comment]
> Well currently with Wither Sickness we don't want to make it too extreme since if you introduce new things it could ruin game progress, I will say I think your ideas are pretty good, having usernames scramble could be really fun, and an indicator that death is imminent could be a good idea, but with weakness/slowness it'd make game progression a nightmare, and we don't mind the yapping, giving us a good description is always good to do

All of this feature would be a game rule, probably disabled by default, just added to the mod for a more fun/immersive experience, or for those that just want the challenge of it. There would also be subgamerules for the game rule like being able to disable slowness and weakness effects from occurring at wither sickness III. there’s some more info in the post about subgamerules and stuff.

**Comment by kun4x** [comment]
> All of this feature would be a game rule, probably disabled by default,

honestly this would solve the big problem i had with this suggestion, its way too hard and wither sickness is already annoying to deal with so it being optional is great, as long as its in the config/gamerule and disabled by default this is a great suggestion, 
its more accurate to story mode but still different and unique

**Comment by nonamecrackers2** [comment]
These are interesting ideas and while I don't necessarily dislike them, adding this stuff would take an immense amount of time and would probably go under appreciated by most people.

**Comment by Wolfzul** [comment]
> These are interesting ideas and while I don't necessarily dislike them, adding this stuff would take an immense amount of time and would probably go under appreciated by most people.

Alright I understand. I know a bit about coding and how it’s a lot of work, so I understand your decision. I really love and appreciate your work, so if you do come around to adding even a little bit of the suggestion maybe in the evolution part 2 update, I’d appreciate it a lot. Thanks 👍

**Comment by Frey4SH** [comment]
I like creating little stories with my minecraft worlds, and I usually play multiplayer, so honestly this sounds like this was suggested for me.
I love how chaotic the username/skin swapping and/or censoring is as well, you're probably gonna expect to not be entirely serious when playing with friends anyways, and if you're getting to that point of wither sickness you've probably got some sort of hideout/bunker as well since you haven't died, so it can cause tension in an otherwise more calm moment or maybe even be comedic depending on what's going on at the time.
I would suggest a few things though
Skins/names could possibly an option between them only changing when out of sight (so if the sick player looked away for long enough it would have a chance to change) and a timer (like every five minutes you get blindness for a few seconds and they swap) for preference and possibly performance, 
Voices should just be distorted and not swapped as a player's vc being swapped would probably be too obvious. (like if a player openly mentioned a player other than you when there's only 2 people in the room, or other context based statements)
And less to do with this, but maybe an option to disable the max health drain and purple-pink-magenta effects? Mainly for players who just want the other features and maybe visibility issues, I assume that if this suggestion was implemented the health drain would at least be slowed but having it stop at a certain point or removed would be nice.

At this point too this could be it's own mod as well, as I think an addon mod would be perfect for this so that the player's looking for it could find it and the original mod stays the exact same.

_oops I also yapped a bit-_
But yeah, love this idea!

**Comment by Herobrine982950291** [comment]
Aint reading all that
